packages/core/test/acceptance/destroy_ref_spec.ts

```diff
@@ -64,15 +64,13 @@ describe('DestroyRef', () => {
       expect(onDestroyCalls).toBe(2);
     });
 
-    it('should throw when trying to register destroy callback on destroyed injector', () => {
+    it('should NOT throw when trying to register destroy callback on destroyed injector and call it immediately', () => {
```

Comment:
nit: can we have it lower case ? 
```suggestion
    it('should not throw when trying to register destroy callback on destroyed injector and call it immediately', () => {
```

Reply:
yep, I thought it might make more sense :joy:

Reply:
updated to lowercase